watch.js

```diff
@@ -193,7 +204,7 @@ FirebaseWatcher.prototype.connect = function() {
 					handleReply(self, msg);
 				} else if (msg.d.a == 'd' || msg.d.a == 'm') {
 					if (self.state == IGNORE_NEXT_STREAM) {
-						self.state = NORMAL;
+						self.log('msg, ' + self.watchReqs + ' to go');
 					} else {
 						handleUpdate(self, msg.d.b.p, msg.d.b.d);
```

Comment:
Unrelated to this PR, but I'm not very happy with these `d`, `b`, `d` thingies. It's all very cryptic although there are some comments about the meaning of the letter.